src/nncf/torch/function_hook/hook_executor_mode.py

```diff
@@ -294,23 +273,16 @@ def get_current_executed_op_name(self, fn_name: str) -> str:
         """
         Get the name of the current operation being executed.
 
+        Note: should be called only ones because it updates op_calls.
```

Comment:
```suggestion
        Note: should be called only once because it updates op_calls.
```
IMO, this warning can be easily overlooked